### PR TITLE
Partial cleanup of JanusGraphTest

### PR DESCRIPTION
**Pull request by farodin91**
Part of #1015

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: janusgraph-test/src/main/java/org/janusgraph/TestCategory.java [hunk 1]
```diff
@@ -21,4 +21,5 @@
     public static final String UNORDERED_KEY_STORE_TESTS = "UNORDERED_KEY_STORE_TESTS";
     public static final String MEMORY_TESTS = "MEMORY_TESTS";
     public static final String PERFORMANCE_TESTS = "PERFORMANCE_TESTS";
+    public static final String CELL_TTL_TESTS = "CELL_TTL_TESTS";
```
Comment (mad) [inline janusgraph-test/src/main/java/org/janusgraph/TestCategory.java]: @farodin91 Hi, could you clarify some issues for me?

Why this category introduced? And why it is disabled by default?

I think `features.hasCellTTL()` more convenient when I run test from IDE, now IDE ignore this annotation and trying run these test on non supported backend

Comment (farodin91) [reply janusgraph-test/src/main/java/org/janusgraph/TestCategory.java]: I did this as a performance improvement. Before this change, maven would run start and stop phase for each test without doing anything.

I would like to move this behaviour into an extension for Junit5. To use feature flag from the backend directly to activate tests.

Comment (mad) [reply janusgraph-test/src/main/java/org/janusgraph/TestCategory.java]: Thanks

But `features.hasCellTTL()` options non conflict with TestCategory flag, this is second check for backend compatibility. May be restore part of code for simplify running test from IDE?

Comment (farodin91) [reply janusgraph-test/src/main/java/org/janusgraph/TestCategory.java]: I'm nearly finished the implementation to abort test based on backend features. https://github.com/farodin91/janusgraph/tree/feature-flag

**Pull request merged**